### PR TITLE
chore(codegen): always choose between Java- and Scala testdata

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM, great to make these cleanups now, before we duplicate to event sourced and replicated entities

File: codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala [hunk 1]
```diff
@@ -298,13 +298,13 @@ object ActionServiceSourceGenerator {
...
         |  override final def serviceDescriptor: Descriptors.ServiceDescriptor =
-        |    ${service.fqn.parent.name}Proto.javaDescriptor.findServiceByName("${service.fqn.protoName}")
+        |    ${typeName(service.descriptorObject)}.javaDescriptor.findServiceByName("${service.fqn.protoName}")
         |
         |  override final def newHandler(context: ActionCreationContext): ${service.handlerName} =
         |    new ${service.handlerName}(actionFactory(context))
         |
         |  override final def additionalDescriptors: immutable.Seq[Descriptors.FileDescriptor] =
-        |    ${service.fqn.parent.name}Proto.javaDescriptor ::
+        |    ${typeName(service.descriptorObject)}.javaDescriptor ::
```
Comment (octonato) [inline codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala:307]: I think I just fixed this and is merged already.

**Review by octonato — APPROVED** [review]
LGTM

**Comment by octonato** [comment]
If I didn't say it before, here it is. I find  that the codegen is getting neater each day. I like the introduction of the methods generateManaged, generateUnmanaged and generateManagedTest. Also the fact that we are improving the model and not copying past or doing too many low level data manipulations all over the place.

File: codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala (hunk 1 above)
Comment (raboof) [reply codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala:300]: you might be thinking of https://github.com/lightbend/akkaserverless-java-sdk/pull/475#discussion_r714766515 , which seems to be failing that PR and motivated me to do this cleanup :)

Comment (octonato) [reply codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ActionServiceSourceGenerator.scala:300]: yeah, thanks. I will rebase #475

**Pull request merged**